lib/sidekiq/api.rb

```diff
@@ -355,8 +355,12 @@ def display_args
       # Unwrap known wrappers so they show up in a human-friendly manner in the Web UI
       @display_args ||= case klass
                 when /\ASidekiq::Extensions::Delayed/
-                  safe_load(args[0], args) do |_, _, arg|
-                    arg
+                  safe_load(args[0], args) do |_, _, arg, kwarg|
+                    if kwarg.present?
```

Comment:
`present?` is a Rails method, right? Needs to work without Rails.

Reply:
Thanks @mperham . I have updated it. Please have a look.